ex_webrtc/integration_test/test_videoroom/test/integration/basic_test.exs

```diff
@@ -56,15 +56,15 @@ defmodule TestVideoroom.Integration.BasicTest do
           {:after_warmup, browsers} ->
             Enum.each(browsers, fn {browser_id, stats_list} ->
               Enum.each(stats_list, fn stats ->
-                assert length(stats) == browser_id
```

Comment:
fyi: Now each track is a separate stream, so the expected number of streams is double the number of peers